debian/changelog

```diff
@@ -1,10 +1,11 @@
-cloud-init (23.3.2-0ubuntu0~23.04.1) UNRELEASED; urgency=medium
-
+cloud-init (23.3.2-0ubuntu1~20.04.1) UNRELEASED; urgency=medium
```

Comment:
Not critical as we'll be new-upstream-snapshotting 23.3.3 in a few mins but this version is off 
```suggestion
cloud-init (23.3.2-0ubuntu1~23.04.1) UNRELEASED; urgency=medium
```

Reply:
Not critical, but I fixed it for my own sanity and future reference.

Reply:
Update: Per latest sidechannel conversation and docs, this version should have been `23.3.2-0ubuntu0~23.10.1`